src/merge.py

```
@@ -83,68 +83,68 @@ def validate_2fa(api: PyiCloudService) -> bool:
 
     return status
 
 def get_datetime(dt:datetime) -> datetime:
     return datetime(dt.year, dt.month, dt.day, dt.hour, dt.minute, tzinfo=dt.tzinfo)
 
-def get_from_list(items:list, value:str):
+def get_from_list(items, value:str):
     try:
         return_value = items.get(value)
-    except:
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)